app/harvesters/Itms/accounting_documents_harvester.rb

```diff
@@ -0,0 +1,92 @@
+require 'harvester_utils/downloader'
+
+class Itms::AccountingDocumentsHarvester
```

Comment:
nesuhlasim, ze toto je celkom mala abstrakcia. je to rozhodovanie o buducom designe. ak mas zaujem, poslem ti ukazku, ako vyzerala prva verzia tohto harvestora, respektive existoval len jeden harvestor, ktory sa nasledne pouzil v kompozicii s dalsimi objektmi. ak aktualne pouzijes co i len malu dedicnost, prediktujes tym design.. ak teda pride po tebe nejaky junior programator, vezme to ako "patter" a vobec nebude premyslat nad tym, ze existuje aj idealnejsia cesta. tj. niesom presvedceny o dedicnosti v tomto pripade, preto mi pride copy & paste idealnejsia cesta.